src/js/lib/monster.ui.js

```diff
@@ -810,9 +870,64 @@ define(function(require) {
 					closeBtnText = '<i class="monster-dialog-titlebar-close"></i>';
 					break;
 			}
-			dialog.siblings().find('.ui-dialog-titlebar-close').html(closeBtnText);
+			$dialogBody.siblings().find('.ui-dialog-titlebar-close').html(closeBtnText);
+
+			// Prepare scrollable container
+			if (scrollableContainer) {
+				if (scrollableContainer instanceof $) {
+					$scrollableContainer = scrollableContainer;
+				} else if (_.isElement(scrollableContainer)) {
+					$scrollableContainer = $(scrollableContainer);
+				} else {
+					$scrollableContainer = $dialogBody.find(scrollableContainer);
+				}
+			}
+			// If scrollable content was not specified, or is not a dialog's descendant,
+			// then default to dialog's body as scrollable content
+			if (!scrollableContainer || _.isEmpty($scrollableContainer) || !$.contains(getFullDialog().get(0), $scrollableContainer.get(0))) {
+				$scrollableContainer = $dialogBody;
+			}
+
+			// Make container scrollable
+			$scrollableContainer.css({
+				overflow: 'auto'
+			});
+
+			// Set initial sizes
+			setDialogSizes();
+
+			// Set event handlers
+			$('input', content).keypress(function(e) {
+				if (e.keyCode === 13) {
+					e.preventDefault();
+					return false;
+				}
+			});
+			$window.on('resize', windowResizeHandler);
+
+			// Check periodically for window width changes, or dialog width/position changes,
+			// to center dialog
+			resizeCheckIntervalId = setInterval(function() {
```

Comment:
I agree. I was thinking to leave it in case there were dialogs whose width changed depending on the active tab, but I haven't seen one like that so far.